.eslintrc.json

```diff
@@ -11,6 +11,7 @@
         "warn",
         { "args": "none" }
     ],
+    "no-redeclare": "off",
```

Comment:
We still want this for the other sub-repos and any node scripts in `/scripts`. Can you move this to the client `.eslintrc.json`?